packages/wrangler/src/pages/download-config.ts

```diff
@@ -0,0 +1,226 @@
+import { writeFile } from "node:fs/promises";
+import TOML from "@iarna/toml";
+import chalk from "chalk";
+import { fetchResult } from "../cfetch";
+import { getConfigCache } from "../config-cache";
+import { FatalError } from "../errors";
+import { logger } from "../logger";
+import * as metrics from "../metrics";
+import { printWranglerBanner } from "../update-check";
+import { requireAuth } from "../user";
+import { PAGES_CONFIG_CACHE_FILENAME } from "./constants";
+import type { RawEnvironment } from "../config";
+import type {
+	CommonYargsArgv,
+	StrictYargsOptionsToInterface,
+} from "../yargs-types";
+import type { PagesConfigCache } from "./types";
+import type { Project } from "@cloudflare/types";
+
+// TODO: fix the Project definition
```

Comment:
Ticket?

Reply:
It depends on [SHIP-8984](https://jira.cfdata.org/browse/SHIP-8984)